simulation/simtypes/app.go

```diff
@@ -15,6 +16,7 @@ type App interface {
 	AppCodec() codec.Codec
 	GetAccountKeeper() AccountKeeper
 	GetBankKeeper() BankKeeper
+	GetStakingKeeper() stakingkeeper.Keeper
```

Comment:
Is there any reason you called the entire Keeper here instead of defining the used interfaces below?

Reply:
Its because we already implemented stakingKeeper in [modules.go](https://github.com/osmosis-labs/osmosis/blob/main/app/modules.go#L275) for ibc stuff and i didn't wanted to break any of that

Reply:
And i believe we use *OsmosisApp interface in [osmosis_helper.go ](https://github.com/osmosis-labs/osmosis/blob/main/tests/simulator/osmosis_helper.go) which was error due to different return types

Reply:
This checks out, thanks for double checking :)